### PR TITLE
Add  Data Collection GUI

### PR DESCRIPTION
**Pull request by jonathanm-12**
This GUI features tasks that instruct the user to perform different actions.
~~It still needs a back-end that collects data.~~

### CODE REVIEW

**Review by mitch-gilmore — APPROVED** [review]
Reviewed this in person and it looked functionally correct. Looks like getting the hardware to have good readings will take some time still, but software seems mature.

**Comment by jonathanm-12** [comment]
~~@mgilmore42 Imagery option doesnt work yet, small fix but I will re-open this once I change it.~~

File: Data Collection/taskwindow.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+    Tasks(canvas, window)
+
+    board.start_stream()
+    window.mainloop()
+
+    data = board.get_board_data()[channels]
+    board.release_session()
+    df = pd.DataFrame(data)
+
```
Comment (Bshortall) [inline Data Collection/taskwindow.py:82]: Need to save the timestamps of when each action is displayed so we know how to label the data

Comment (jonathanm-12) [reply Data Collection/taskwindow.py:82]: see new commit

**Review by Bshortall — APPROVED** [review]
looks good

**Pull request merged**